netlify.toml

```diff
@@ -0,0 +1,7 @@
+[[plugins]]
```

Comment:
We might actually want to turn this off now that we've "completed" the cutover as it really blows our build budget at Netlify by about extra 2 or 3 minutes... or maybe we reach out to them and see if they could hook us up with more time for helping them beta test this...

Reply:
better if we can mooch more time

we can suffice with google search console crawl errors, but that will only be an after-the-fact report, and i'm not sure about notifications

maybe we can run this only on master, or only sporadically?